lms/djangoapps/onboarding_survey/views.py

```diff
@@ -163,14 +173,25 @@ def organization(request):
     otherwise, a form is populated form the POST request and then is
     saved. After saving the form, user is redirected to dashboard.
     """
+
+    def set_survey_complete(extended_profile):
+        extended_profile.are_surveys_complete = True
+        extended_profile.save()
+        return True
```

Comment:
what is the reason for returning `True` here?

Reply:
Removed.